release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -1269,53 +1279,53 @@ default interface for the operating system is selected based on common template
 
 * The tile description variable was incorrectly set to be the CRD description appended with the CSV description. This caused the tile descriptions to be wrong. The tile descriptions are now back to the original value and the appended value is now moved to its own variable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1814639[*BZ#1814639*])
 
 * The `eventSources` API Group is updated to the latest supported API Group, `sources.knative.dev`. This update allows sources generated by the new API Group to be recognized in the *Topology* view of the web console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1836805[*BZ#1836805*])
 
-* With the release of Red Hat OpenShift Serverless 1 Serverless Operator version 1.7.1, the Operator is generally available. The Tech Preview badge in the Developer perspective of the web console has been removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1827042[*BZ#1827042*])
+* With the release of Red Hat OpenShift Serverless 1 Serverless Operator version 1.7.1, the Operator is generally available. The Tech Preview badge in the *Developer* perspective of the web console has been removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1827042[*BZ#1827042*])
 
 *DNS*
 
-* Previously, CoreDNS metrics were being exposed over an insecure channel within a cluster. Now the proper TLS components and a kube-rbac-proxy sidecar have been added to secure the CoreDNS metrics endpoint and expose CoreDNS metrics over a secure channel. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1809197[*BZ#1809197*])
+* Previously, CoreDNS metrics were being exposed over an insecure channel within a cluster. Now the proper TLS components and a `kube-rbac-proxy` sidecar have been added to secure the CoreDNS metrics endpoint and expose CoreDNS metrics over a secure channel. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1809197[*BZ#1809197*])
 
-* Previously,adding arbitrary taints to nodes could cause problems related to the DNS operator's operand. Now the DNS operator's operand tolerates any taint added to a node. The operand runs on and updates /etc/hosts on all Linux node hosts. *Missing CNI default network* events may be observed when the operand starts on a node that is still initializing, but such errors are transient and can be ignored. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1813479[*BZ#1813479*])
+* Previously, adding arbitrary taints to nodes could cause problems related to the DNS operator's operand. Now the DNS operator's operand tolerates any taint added to a node. The operand runs on and updates `/etc/hosts` on all Linux node hosts. *Missing CNI default network* events may be observed when the operand starts on a node that is still initializing, but such errors are transient and can be ignored. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1813479[*BZ#1813479*])
```

Comment:
Operator should still be capitalized

Reply:
thanks! found several more throughout :+1: